packages/services/api/src/modules/organization/resolvers.ts

```diff
@@ -13,7 +13,14 @@ import { Logger } from '../shared/providers/logger';
 import type { OrganizationModule } from './__generated__/types';
 import { OrganizationManager } from './providers/organization-manager';
 
-const OrganizationNameModel = z.string().min(2).max(50);
+const OrganizationNameModel = z
+  .string()
+  .min(2)
+  .max(50)
+  .regex(
+    /^([A-Za-z]|[0-9]|_| )+$/,
+    "Only characters 'A-Z', 'a-z', '0-9', '_', '-' and ' ' are allowed.",
```

Comment:
Let's allow `.`, `&`, `-`, `,` and `/`. I went through existing organizations and that's what top 100 orgs have in their names.

Reply:
feel free to take over this PR